tests/test_utils.py

```diff
@@ -0,0 +1,14 @@
+from models import PullRequest
+import json
+
+class MockPullRequest(PullRequest):
+    def __init__(self, labels_json_file):
+        PullRequest.__init__(self)
+        self.labels_json = json.load(open(labels_json_file))
```

Comment:
you aren't closing this file ever. You should do:
```python
with open(labels_json_file) as f:
    self.labels_json = json.load(f)
```
so that the file handler closes once it has been read